lib/assisted_workflow/addons/pivotal.rb

```diff
@@ -1,6 +1,6 @@
 require "assisted_workflow/exceptions"
 require "assisted_workflow/addons/base"
-require 'pivotal_tracker'
+require 'tracker_api'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.